src/shell/index.js

```diff
@@ -9,6 +9,8 @@ import { Router } from "react-router-dom";
 import { ThemeProvider, createTheme } from "@mui/material/styles";
 import { theme } from "@zesty-io/material";
 
+import "chart.js/auto";
```

Comment:
Why does this happen in the shell?

Reply:
Ah this is required for v3 and added it here to verify all charts. Will move